Makefile

```diff
@@ -133,6 +133,7 @@ GINKGO_NOCOLOR ?= false
 ARTIFACTS ?= $(ROOT_DIR)/_artifacts
 E2E_CONF_FILE ?= $(ROOT_DIR)/test/e2e/config/e2e_conf.yaml
 E2E_OUT_DIR ?= $(ROOT_DIR)/test/e2e/_out
+export E2E_OUT_DIR
```

Comment:
To store logs, instead of exporting this I would prefer to use `artifactFolder` [already exists in the e2e](https://github.com/metal3-io/cluster-api-provider-metal3/blob/e735768beba749e46b43c143e71c8fad6161a52f/test/e2e/e2e_suite_test.go#L42)

Reply:
Thanks. I'll use that one

Reply:
you have also to remove the export here